src/tabs/pid_tuning.html

```diff
@@ -656,7 +656,38 @@
             <div class="subtab-rates" style="display: none;">
                 <div class="clear-both"></div>
                 <div class="cf_column">
-                    <div class="note maxRateWarning">
+                
+                    <div class="gui_box rates_type pid_tuning">
+                        <table class="cf">
+                            <thead>
+                                <tr>
+                                    <th>
+                                        <div>
+                                            <div i18n="pidTuningRatesType" style="float:left;"></div>
```

Comment:
We try to not add more `style` tags to the html. If you can, please add a class here or add/modify some style in the CSS.

Reply:
All the style i added is in css, this is from old code that i copied. Many problems even found by sonar are old code that was triggered by my modification. In that particular case i need to modify all the old code that contains `style="float:left;"`?

Reply:
Not all of it. But one tenet of collaborative software development is 'always leave the code at least as clean as you found it, _or cleaner_'. And in this case, since you touched this code, you should also clean it up to the level issues detected by Sonar (that aren't false positives) are resolved. If everybody does this, the overall quality of the code will improve over time.